From 7adc2ec491c181dda31c56bc096cad77ba9d05fb Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey.doherty@lispysnake.com>
Date: Thu, 27 Aug 2020 17:45:45 +0100
Subject: [PATCH] partition/makefs: Include missing sys/file.h header

This file must be included on non-glibc systems to ensure
the `LOCK_EX` definition is available.

Signed-off-by: Ikey Doherty <ikey.doherty@lispysnake.com>
---
 src/partition/makefs.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/partition/makefs.c b/src/partition/makefs.c
index 97f50c9..33ede22 100644
--- a/src/partition/makefs.c
+++ b/src/partition/makefs.c
@@ -1,6 +1,7 @@
 /* SPDX-License-Identifier: LGPL-2.1+ */
 
 #include <fcntl.h>
+#include <sys/file.h>
 #include <sys/prctl.h>
 #include <sys/stat.h>
 #include <sys/types.h>
-- 
2.27.0

